app/routes/room.$roomId.player.tsx

```diff
@@ -28,7 +32,11 @@ export async function action({ request, params }: ActionFunctionArgs) {
   }
 
   const type =
-    request.method === "POST" ? ActionType.Join : ActionType.ChangeName;
```

Comment:
can you change this to a `switch (request.method)` now instead of multiple ternary operators? the cute/annoying way to do this without reassignment is something like:

```ts
const type = () => {
    switch (request.method) {
        case "POST":
            return ActionType.Join;
        // ...
}();
```